src/routes/admin.js

```diff
@@ -0,0 +1,35 @@
+import express from 'express'
+import path from 'path'
+import * as templateTask from '../database/queries/template_task'
+import cors from 'cors'
+const router = express.Router()
+
+router.get('/template_task/get_template_tasks', function(req, res, next){
+  let templateTasks = { mentor: [], noob: [] }
+  templateTask.getAll()
+  .then( results => {
+    for(let task of results){
```

Comment:
I would recommend using the `groupBy` function(available in `lodash/groupBy` when you want to split an array by a condition. The `for` statement would be reduced to a single `groupBy` function. This would looks something like this -
```javascript
import groupBy from 'lodash/groupBy'

const templateTasks = groupBy(results, task => task.user_role)
res.json(templateTasks)
```